src/clj/util/forms.clj

```diff
@@ -0,0 +1,37 @@
+(ns util.forms)
+
+
+(defn as-coll
+  "Wraps the parameter in a vector if it is not already a collection."
+  [coll-or-single]
+  (cond
+    (empty? coll-or-single) []
+    (coll? coll-or-single) coll-or-single
+    :else [coll-or-single]))
+
+
+(defn validate-form-data
```

Comment:
Could you specify the input functions more? It's nice that you've written down, how the input is supposed to look like, but I think with either a spec or an example call, it would be more clear, on how to use this function.